msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1454,6 +1452,14 @@ When you use these install and uninstall events, there are some instances where
 
 It's important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you're using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you don't choose to handle.
 
+## Handling errors in conversation events
+
+When the bot encounters an error while handling different events or activities, you should not post random messages directly to the conversation which has no meaningful context like below:
+
+:::image type="content" source="../../../assets/images/error-msg.PNG" alt-text="Error message to the conversation with no meaningful context.":::
+
+Send the meaningful messages in conversations which is specific to the errors for better user experience.
```

Comment:
I can't see reference link for below:
[Handle unexpected errors](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-howto-handle-user-interrupt?view=azure-bot-service-4.0&tabs=csharp#handle-unexpected-errors)